### PR TITLE
Clarify docs surrounding ECK Enterprise licensing

### PR DESCRIPTION
**Pull request by aliciascott**
Adding some additional details to free up confusion surrounding Enterprise licensing

Namely, adding details to contents of orchestration.json file and why the `_license` api returns as `Platinum`.

### CODE REVIEW

**Comment by aliciascott** [comment]
@pebrc @idanmo not sure if I created this against the correct branch, please review and merge changes as you agree to.

File: docs/operating-eck/licensing.asciidoc [hunk 1]
```diff
@@ -48,7 +48,7 @@ At the end of the trial period, the Platinum and Enterprise features operate in
 [float]
 [id="{p}-add-license"]
 == Add a license
-If you have a valid Enterprise subscription or a trial license extension, you will receive a license as a JSON file.
+If you have a valid Enterprise subscription or a trial license extension, you will receive a license as a JSON file. This file contains the ECK orchestration(Enterprise) license which enables ECK features, as well as individual Platinum licenses for your actual Elasticsearch cluster(s).
```
Comment (pebrc) [inline docs/operating-eck/licensing.asciidoc:51]: That's not fully correct. The orchestration Enterprise license contains cluster-level Enterprise licenses as well as Platinum licenses for older Elasticsearch version.s

**Review by pebrc — COMMENTED** [review]
There are a lot of unrelated changes in this PR that we would need to get rid of in order to merge this I think.

File: docs/operating-eck/licensing.asciidoc (hunk 1 above)
Comment (aliciascott) [reply docs/operating-eck/licensing.asciidoc:51]: Ok thanks for the note, I'll get that updated

**Comment by aliciascott** [comment]
Ok, I wondered why it was trying to merge other changes, let me close this out and open a new one against master :)

**Pull request closed without merging**